mcp_zammad/__main__.py

```
@@ -1,27 +1,53 @@
+        pass
+    except ValueError as e:
+        # Configuration error, log w/o stack trace
+        logger.error("Configuration error: %s", e)  # noqa: TRY400
+        sys.exit(1)
+    except RuntimeError as e:
+        # Runtime error, log w/o stack trace
+        logger.error("Runtime error: %s", e)  # noqa: TRY400
+        sys.exit(1)
+    except Exception as e:
+        # Unexpected internal error, include full stack trace
+        logger.error("Internal error: %s", e, exc_info=True)
```

Ruff (0.15.12)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)